cmd/context/create.go

```diff
@@ -380,6 +380,28 @@ func (c ContextCommand) getUserContext(ctx context.Context, ns string) (*types.U
 				return nil, fmt.Errorf(oktetoErrors.ErrCorruptedOktetoContexts, config.GetOktetoContextsStorePath())
 			}
 		}
+
+		kubetoken, err := client.Kubetoken().GetKubeToken(okteto.Context().Name, okteto.Context().Namespace)
+		if err != nil {
+			if utils.LoadBoolean(model.OktetoUseStaticKubetokenEnvVar) {
+				oktetoLog.Info("You are using static kubernetes tokens because you set '%s', however the feature is not available and okteto CLI will be using a static token", model.OktetoUseStaticKubetokenEnvVar)
+			}
+			oktetoLog.Debug("Dynamic kubernetes tokens not available: falling back to static token")
+			return userContext, nil
+		}
+
+		if utils.LoadBoolean(model.OktetoUseStaticKubetokenEnvVar) {
+			oktetoLog.Info("You are using static kubernetes tokens because you set '%s'. This feature will be removed in the future. We recommend using dynamic kubernetes tokens, to know more check out out documentation: https://www.okteto.com/docs", model.OktetoUseStaticKubetokenEnvVar)
```

Comment:
@rlamana We need to evaluate which documentation we are going to generate and see if this is the right call to action.

Also, I think these messages **won't** be displayed to the user. Is that what we want? Should they be warnings?

Reply:
I've changed it to warning and pinged @rlamana too so we can get approval from product

Reply:
discussed with @rlamana and agreed to remove the link to docs until it's available